.eslintrc

```diff
@@ -3,7 +3,8 @@
   "extends": [
     "eslint:recommended",
     "fliplet",
-    "plugin:vue/essential"
+    "plugin:vue/essential",
+    "Modernizr"
```

Comment:
Why is this extending the ESLint config with `Modernizr` as opposed to adding `"Modernizr": true` to the `globals` object? I assume you're doing this because `Modernizr` is not globally available.

Reply:
@tonytlwu No, `Modernizr` is globally available, and you are right I'll move it to the `globals` object.

Reply:
@tonytlwu Fixed